charts/substra-backend/templates/deployment-scheduler.yaml

```diff
@@ -38,97 +38,57 @@ spec:
-              value: debug
             - name: LEDGER_CONFIG_FILE
               value: /conf/{{ .Values.organization.name }}/substra-backend/conf.json
+            - name: PYTHONUNBUFFERED
+              value: "1"
           volumeMounts:
-            - mountPath: /conf/{{ .Values.organization.name }}/substra-backend
-              name: config
+            - name: config
+              mountPath: /conf/{{ .Values.organization.name }}/substra-backend
               readOnly: true
-            - mountPath: /var/hyperledger/msp/signcerts
-              name: id-cert
-            - mountPath: /var/hyperledger/msp/keystore
-              name: id-key
-            - mountPath: /var/hyperledger/msp/cacerts
-              name: cacert
-            - mountPath: /var/hyperledger/msp/admincerts
-              name: admin-cert
-            - mountPath: /var/hyperledger/tls/server/pair
-              name: tls
-            - mountPath: /var/hyperledger/tls/server/cert
-              name: tls-rootcert
-            - mountPath: /var/hyperledger/tls/client/pair
-              name: tls-client
-            - mountPath: /var/hyperledger/tls/client/cert
-              name: tls-clientrootcert
-            - mountPath: /var/hyperledger/tls/ord/cert
-              name: ord-tls-rootcert
-            - mountPath: /var/hyperledger/admin_msp/signcerts
-              name: admin-cert
-            - mountPath: /var/hyperledger/admin_msp/keystore
-              name: admin-key
-            - mountPath: /var/hyperledger/admin_msp/cacerts
-              name: cacert
-            - mountPath: /var/hyperledger/admin_msp/admincerts
-              name: admin-cert
+            - name: user-cert
+              mountPath: /var/hyperledger/msp/signcerts
+            - name: user-key
+              mountPath: /var/hyperledger/msp/keystore
+            - name: peer-tls-server
+              mountPath: /var/hyperledger/tls/server/pair
+            - name: peer-tls-client
+              mountPath: /var/hyperledger/tls/client/pair
+            - name: orderer-tls-ca
+              mountPath: /var/hyperledger/tls/ord/cert
+            - name: ca
+              mountPath: /var/hyperledger/ca
           resources:
             {{- toYaml .Values.celeryworker.resources | nindent 12 }}
       volumes:
       - name: config
         configMap:
-          name: {{ include "substra.fullname" . }}-backend
-      - name: fabric
-        configMap:
-          name: {{ $.Values.secrets.fabricConfigmap }}
-      - name: id-cert
-        secret:
-          secretName: {{ $.Values.secrets.cert }}
-      - name: id-key
-        secret:
-          secretName: {{ $.Values.secrets.key }}
-      - name: cacert
-        secret:
-          secretName: {{ $.Values.secrets.caCert }}
-      - name: tls
-        secret:
-          secretName: {{ $.Values.secrets.tls }}
-      - name: tls-rootcert
+          name: {{ include "substra.fullname" . }}-server
+          readOnly: true
```

Comment:
Volumes doesn't implement readOnly by themselves, it's specified during the mount: 
```
Error: validation failed: error validating "": error validating data: ValidationError(Deployment.spec.template.spec.volumes[0].configMap): unknown field "readOnly" in io.k8s.api.core.v1.ConfigMapVolumeSource
```

Reply:
thanks